__tests__/compilers/compatability.test.tsx

```diff
@@ -320,4 +320,14 @@ This is an image: <img src="http://example.com/#\\>" >
     `,
     );
   });
+
+  it('trims whitespace surrounding phrasing content (emphasis, strong, etc)', () => {
+    const md = `** bold ** and * italic * and *** bold italic ***`;
```

Comment:
I'd add the alternate `_` emphasis syntax to this test (and maybe a few extra weird spaces) just to clarify that this fix should work for both:

```suggestion
    const md = `** bold ** and _ italic _ and ***   bold italic ***`;
```